### PR TITLE
Meta.security.system should be specified for LOA codes

### PR DESCRIPTION
**Pull request by masnick**
This PR follows guidance we got from [Grahame on Zulip](https://chat.fhir.org/#narrow/stream/284830-smart.2Fhealth-cards/topic/Bundle.20Format/near/235575616) related to our use of `Meta.security` for LOA:

> you have a value set. you just need to specify to fill out ```Meta.security.system``` with the value ```http://terminology.hl7.org/CodeSystem/loa```.

This PR changes our examples to use `http://terminology.hl7.org/CodeSystem/loa` for `Meta.security.system`, and removes the corresponding note on the Conformance page about the validator error related to this.

Issuers SHOULD populate `Meta.security.system` when populating `Meta.security.code` to avoid a warning like `A code with no system has no defined meaning. A system should be provided` when validating.

We are working with HL7 Vocab to ensure `http://terminology.hl7.org/CodeSystem/loa` is the correct code system URI. This PR will remain in draft until we have that confirmed.

### CODE REVIEW

**Comment by JohnMoehrke** [comment]
The LOA codes already exist in HL7 in this valueset https://terminology.hl7.org/ValueSet-v3-SecurityTrustObservationValue.html
Thus your codesystem would be http://terminology.hl7.org/CodeSystem/v3-ObservationValue

**Comment by masnick** [comment]
There was a conversation on Zulip about this that I wasn't aware of -- it sounds like people on the `Security and Privacy` stream decided we do need a new CodeSystem because we needed to add new "intermediate codes" to fill the space between IAL1 and IAL2, which is needed for our use case.

I've logged https://jira.hl7.org/browse/UP-197 to get `http://terminology.hl7.org/CodeSystem/loa` added to UTG.

**Comment by JohnMoehrke** [comment]
I agree adding a code to the HL7 managed codesystem is appropriate. (Adding a code to a codesystem managed by NIST would not be appropriate for us to do). But this is clearly a need driven code, and the current LOA codes are in an HL7 managed codesystem.

**Comment by jmandel** [comment]
> Defining URL: http://hl7.org/fhir/identity-assuranceLevel

> I'll note that neither this nor the http://terminology.hl7.org/CodeSystem/v3-ObservationValue code systems have been updated to include 800-63-3 codes.

And of course 800-63-3 doesn't have our "1.5"ish concepts either.

**Comment by masnick** [comment]
Until HL7 can add these codes to a code system, we are going to use `https://smarthealth.cards/ial` as the URI, which will be active once smart-on-fhir/health-cards#142 is merged in. This could also be the permanent URI if HL7 is ok with this code system being externally defined. 

This will look like:

```
...
"resource": {
   "resourceType": "Immunization",
   "meta": {
     "security": [{"system": "https://smarthealth.cards/ial", "code": "IAL1.2"}]
   },
   ...
```

If we eventually move to a code system defined within HL7, we will want to make sure that:

1. The `system` URI resolves when entered into a web browser; and
2. The contents of the page that displays will not be confusing to a technical person without FHIR experience.

We can then redirect `https://smarthealth.cards/ial` to the HL7 page defining the code system.

The reason for making sure the URI resolves in a browser and is as easy to interpret as possible is that our implementers will include software developers who do not have prior experience with FHIR, and may encounter JSON resources "in the wild" without the context of the FHIR IG. These developers may not know to look for the FHIR IG, or understand what a "code system" is -- instead, they may just navigate to the value of `meta.security.system` in a browser as this looks like a URL, and their expectation will be that (1) this URI will resolve; and (2) the contents of the page will clearly describe what `IAL1.2` means in the context of this code system.

We also want to support patients who wish to examine the contents of their SMART Health Cards as much as possible. This follows from the [VCI key design principles](https://vci.org/about) of "flexible and equitable" and "transparent". While we can't provide patients with the technical ability to interpret JSON or the defining specs, we can lower the barrier to entry as much as possible.

We therefore want to avoid using a code system defining URI that doesn't resolve in a browser, or resolves to a code system page like http://terminology.hl7.org/CodeSystem/v3-ObservationValue (which includes hundreds of unrelated codes and doesn't provide a mechanism for cleanly and clearly explaining this particular code system).

File: input/pagecontent/index.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ The [VaccinationCredentialPatient] and [Covid19LaboratoryBundle]/[InfectiousDise
 
 ```json
 "meta": {
-  "security": [{"code": "IAL1"}]
+  "security": [{"code": "IAL1", "system": "http://terminology.hl7.org/CodeSystem/loa"}]
```
Comment (p9g) [inline input/pagecontent/index.md:109]: Need to use the new URI for system

Comment (p9g) [reply input/pagecontent/index.md:109]: Also change IAL1 to one of the other values in that value set

Comment (nkarip) [reply input/pagecontent/index.md]: updated, thanks!

**Comment by p9g** [comment]
Note that https://terminology.hl7.org/2.1.0/CodeSystem-v3-ObservationValue.html references 800-63-2, but we reference 800-63-3. 

800-63-3 supersedes NIST Special Publication 800-63-2 according to https://pages.nist.gov/800-63-3/sp800-63-3.html

**Pull request merged**